src/coffee.js

```diff
@@ -0,0 +1,376 @@
+const coffee = [
+  {
+    name: "Aynu",
+    profile: "clean",
+    //	characteristic: "sweet", // or "acidic" or "round"
```

Comment:
@DEJENERUTA please only edit this one file, don't edit any other files.

We're missing 3 pieces of data for each coffee:
- the `characteristic`, which is `sweet` or `acidic` or `round`
- the `url` which points to IR's website to the exact coffee
- the `picture` which is a picture of the coffee, coming from IR's website